sql/telemetry_derived/fxa_users_services_daily_v1/query.sql

```diff
@@ -0,0 +1,114 @@
+CREATE TEMP FUNCTION
+  udf_mode_last(list ANY TYPE) AS ((
+    SELECT
+      _value
+    FROM
+      UNNEST(list) AS _value
+    WITH
+    OFFSET
+      AS
+    _offset
+    GROUP BY
+      _value
+    ORDER BY
+      COUNT(_value) DESC,
+      MAX(_offset) DESC
+    LIMIT
+      1 ));
+--
+CREATE TEMP FUNCTION
+  udf_contains_tier1_country(x ANY TYPE) AS ( --
+    EXISTS(
+    SELECT
+      country
+    FROM
+      UNNEST(x) AS country
+    WHERE
+      country IN ( --
+        'United States',
+        'France',
+        'Germany',
+        'United Kingdom',
+        'Canada')) );
+  --
+  -- This UDF is also only applicable in the context of this query.
+CREATE TEMP FUNCTION
+  udf_contains_registration(x ANY TYPE) AS ( --
+    EXISTS(
+    SELECT
+      event_type
+    FROM
+      UNNEST(x) AS event_type
+    WHERE
+      event_type IN ( --
+        'fxa_reg - complete')) );
+  --
+WITH
+  base AS (
+    SELECT
+      * REPLACE(
+        -- cert_signed is specific to sync, but these events do not have the
+        -- 'service' field populated, so we fill in the service name for this special case.
+        IF(service IS NULL AND event_type = 'fxa_activity - cert_signed', 'sync', service) AS service)
```

Comment:
@irrationalagent This is an attempt to address your concern about cert_signed events not being attributable to sync. This inserts 'sync' as the service at the point where we populate the daily table, and then would flow through to last_seen.

Reply:
yes, this looks like a good solution. if in the future we fix things on FxA's side to start populating service on this event then we can remove this (though it won't break anything if we do make the fix)